deploy/runtools/firesim_topology_with_passes.py

```diff
@@ -363,10 +376,13 @@ def phase_one_passes(self) -> None:
     def pass_build_required_drivers(self) -> None:
         """ Build all simulation drivers. The method we're calling here won't actually
         repeat the build process more than once per run of the manager. """
-        servers = self.firesimtopol.get_dfs_order_servers()
 
-        for server in servers:
-            server.get_resolved_server_hardware_config().build_sim_driver()
+        def build_drivers(servers: List[FireSimServerNode]) -> None:
+            for server in servers:
+                server.get_resolved_server_hardware_config().build_sim_driver()
+
+        servers = self.firesimtopol.get_dfs_order_servers()
+        execute(build_drivers, servers, hosts=['localhost'])
```

Comment:
Why does this need to be put behind `fabric.execute(hosts=['localhost'])`?

Reply:
We need it to be executed via run() instead of local(), so that the environment is set correctly (because we source sourceme-f1-manager.sh when we do this build). Otherwise, there are a bunch of error messages from sourceme-f1-manager.sh trying to do conda activate and saying the shell is not properly configured (these currently appear even on firesim main + AWS).

local() by default will use /bin/sh: https://docs.fabfile.org/en/1.8/api/core/operations.html#fabric.operations.local